quic/s2n-quic-core/src/time/mod.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-mod clock;
+pub mod clock;
```

Comment:
I wanted to get at the mock clock implementation. Perhaps there is a better way?

Reply:
`clock` is already exported down below on line 8

```rust
pub use clock::*;
```

Reply:
Thanks!